corehq/apps/users/templates/users/bootstrap5/roles_and_permissions.html

```diff
@@ -43,299 +42,364 @@
+                  <i class="fa-solid fa-eye"></i>
+                  {% trans "View Role" %}
+                </span>
+              </button>
+              <button
+                class="btn btn-outline-danger"
+                data-bind="visible: $data._id && !$data.preventRoleDelete && $root.allowEdit && isEditable,
+                click: $data._id ? $root.setRoleBeingDeleted : null"
+              >
+                <i class="fa-regular fa-trash-can"></i>
+                {% trans "Delete" %}
+              </button>
             </td>
-          {% endif %}
-          <td>
-            <button class="btn btn-outline-primary" data-bind="visible: $data._id, click: $data._id ? $root.setRoleBeingEdited : null">
-                                <span data-bind="if: $root.allowEdit && (!upstream_id() || $parent.unlockLinkedRoles())">
-                                    <i class="fa fa-edit"></i>
-                                    {% trans "Edit Role" %}
-                                </span>
-                                <span data-bind="ifnot: $root.allowEdit && (!upstream_id() || $parent.unlockLinkedRoles())">
-                                    <i class="fa-solid fa-eye"></i>
-                                    {% trans "View Role" %}
-                                </span>
-            </button>
-            <button class="btn btn-outline-danger"
-              data-bind="visible: $data._id && !$data.preventRoleDelete && $root.allowEdit && isEditable,
-                click: $data._id ? $root.setRoleBeingDeleted : null">
-              <i class="fa-regular fa-trash-can"></i>
-              {% trans "Delete" %}
-            </button>
-          </td>
-          {% if is_managed_by_upstream_domain %}
-          <td class="text-center">
-            <span data-bind="if: upstream_id">
-              <i class="fa fa-link"></i>
-            </span>
-          </td>
-          {% endif %}
-        </tr>
+            {% if is_managed_by_upstream_domain %}
+              <td class="text-center">
+                <span data-bind="if: upstream_id">
+                  <i class="fa fa-link"></i>
+                </span>
+              </td>
+            {% endif %}
+          </tr>
         </tbody>
       </table>
       {% if can_edit_roles %}
-        <button class="btn btn-outline-primary"
-                data-bind="click: function () {$root.setRoleBeingEdited($root.defaultRole)}">
+        <button
+          class="btn btn-outline-primary"
+          data-bind="click: function () {$root.setRoleBeingEdited($root.defaultRole)}"
+        >
           <i class="fa fa-plus"></i>
           {% trans "Add Role" %}
         </button>
-        <div data-bind="modal: roleBeingDeleted" tabindex="-1" role="dialog">
-          <div data-bind="with: roleBeingDeleted"
-               class="modal-dialog">
+        <div
+          data-bind="modal: roleBeingDeleted"
+          tabindex="-1"
+          role="dialog"
+          class="modal fade"
```

Comment:
It'd be nicer to review changes like this separate from purely formatting commits, even when they're very small like this.